uvm/impl/com/untangle/uvm/LocalDirectoryImpl.java

```diff
@@ -671,14 +714,25 @@ private void updateRadiusConfiguration(LinkedList<LocalDirectoryUser> list)
 
             if (systemSettings.getRadiusServerEnabled()) {
                 for (LocalDirectoryUser user : list) {
-                    byte[] rawPassword = Base64.decodeBase64(user.getPasswordBase64Hash().getBytes());
-                    String userPassword = new String(rawPassword);
+                    String userPassword;
+                    if(!StringUtils.isBlank(user.getPasswordBase64Hash())){
```

Comment:
Use isNotBlank

Reply:
Also, how about extracting a method for this code, since it seems to be exactly same as line 653.